apm-sniffer/apm-sdk-plugin/kafka-commons/src/main/java/org/apache/skywalking/apm/plugin/kafka/define/KafkaContext.java

```diff
@@ -19,19 +19,30 @@
 
 package org.apache.skywalking.apm.plugin.kafka.define;
 
-public class SpringKafkaContext {
+public class KafkaContext {
 
-    public SpringKafkaContext() {
+    public KafkaContext(String operationName) {
         needStop = false;
+        this.operationName = operationName;
     }
 
     private boolean needStop;
```

Comment:
I rechecked the codes, this seems not necessary. You always `needStop=false`. Could we consider to remove it?

Reply:
fixed